tests/page-objects/EditorPage.ts

```diff
@@ -43,24 +43,34 @@ export class EditorPage {
         await this.addOpposingReviewer();
     }
 
-    private async addPersonToPeoplePicker(picker: Selector): Promise<void> {
+    private async addPersonToPeoplePicker(picker: Selector, number = 1): Promise<void> {
         await t.expect(picker.find('.selected_people_list__item').count).eql(1);
         const addButton = picker.find('.pod__button');
         await t.click(addButton);
+        await this.assertPeoplePickerSearch();
         await t.expect(Selector('.modal__overlay').count).eql(1);
         await t.expect(Selector('.typography__heading--h2').visible).ok();
         await t.expect(Selector('#peoplePickerSearch').visible).ok();
-        const button = Selector('.people-picker__modal_list--item .pod__button');
-        await t.expect(button.visible).ok();
-        await t.click(button);
-        await t.expect(Selector('.people-picker__selected-tabs').child('.people-picker__selected-tab').count).eql(1);
+        const addButtonSelector = Selector('.people-picker__modal_list--item .pod__button');
+
+        const buttons = [];
+        for (let i = 0; i < number; i++) {
+            buttons.push(addButtonSelector.nth(i));
+        }
+
+        for await (const button of buttons) {
+            await t.expect(button.visible).ok();
+            await t.click(button);
+        }
```

Comment:
could this be made into a single loop?
something like 
```ts
for (let i=0; i < number; i++) {
    const button = addButtonSelector.nth(i);
    await t.expect(button.visible).ok();
    await t.click(button);
}
```

Reply:
done